### DISCUSSION TITLE
Stripped down example

### DISCUSSION THREAD

**Discussion by arpitdalal** [epicweb-dev/epic-stack#702] [Ideas]
@kentcdodds Would you be interested in having an example of a stripped-down Epic Stack for just content sites? Meaning no SQLite and Auth code.
If yes, I can submit a PR as I already have a working version.

**Comment by kentcdodds** [comment]
An example would be welcome! Thank you!

**Comment by arpitdalal** [comment]
Thanks Kent!
[Here's](https://github.com/arpitdalal/epic-content-stack) the example.
